src/search.js

```diff
@@ -508,8 +516,11 @@ class Search {
         else {
             var forEachInLine = function(row, startIndex, callback) {
                 re.lastIndex = startIndex;
-                if (multiline) {
+                if (mtSearch) {
                     var pos = mtForward(session, re, row, lastRow);
```

Comment:
can we keep calling function consistent earlier its been called with this.$multiLineForward on line 121.

Reply:
No, because the parameter is different.